Assets/RGLUnityPlugin/Scripts/LidarSensor.cs

```diff
@@ -91,11 +98,15 @@ public class LidarSensor : MonoBehaviour
         private LidarModel? validatedPreset;
         private float timer;
 
+        private Matrix4x4 lastTransform;
```

Comment:
It should be initialized in `Awake()` or `Start()` to avoid a jump at the start of the simulation.